docs/design.ltx

```diff
@@ -0,0 +1,357 @@
+
+\begin{figure}
+\centering
+\begin{tabular}{ | l | l | l | l | l |}
+    \hline
+    -1 & 0 & 0 & 0 & 0\\ \hline
+    0 & 1 & 2 & 1 & 0\\ \hline
+    0 & 3 & 5 & 1 & 0\\ \hline
+    0 & 2 & 2 & 3 & 0\\ \hline
+    0 & 0 & 0 & 0 & -2\\
+    \hline
+\end{tabular}
+\caption{An example of a kernel to be analyzed by automatic scale detection.}
+\end{figure}
+
+When graphed on a Cartesian coordinate system, the scale chosen is the local maximum. As illustrated in figure 2. with an example blob being the center of the flower, the largest and best fitting blob corresponds to the local maximum when graphed on the Cartesian coordinate system.
+
+\begin{figure}
+\centering
+\includegraphics[scale=.5]{blob.jpg}
+\caption{A demonstration of the largest scale blob in the center of the flower corresponding with the local maximum of the curve \cite{ppt}.}
+\end{figure}
+
+One important factor that needs to be considered for this design item is that the values being generated inside the Laplacian of Gaussian kernel can get very, very large \cite{lapofgaus}. It is very important that the environment used when running the automatic scale detection algorithm can support a large range of numbers, both positive and negative. The possibility of overflow is likely and this could very heavily skew the results of blob detection. The development team does not have experience with running this type of algorithm, but support for a minimum of 32-bit numbers is what will be used as a starting point.
+
+% heidi
+\subsection{Classification of AVIRIS data}
+\label{subsec:classification}
+After the data has been processed using feature extraction as detailed in the previous section, the resulting data will be put through a process of classification. The classification will take the AVIRIS images, which are essentially a map of reflected light, and produce an image where each pixel is identified as belonging to a certain class \cite{introarcmap}. This serves the purposes of making the data more readable to the development team and any other person who is using COAL and classifying the data in a useful way to aid in the identifying minerals, identifying mining, and correlating the effects with environmental impact design items. The classes will be types of land surfaces. This includes water (e.g. turbid versus less turbid), vegetation (e.g. dense versus less dense), basaltic lava flow, snow, and any types of minerals found in the image \cite{aviris}. The algorithm used for classification will be a neural network.
```

Comment:
The final sentence in this paragraph ```The algorithm used for classification will be a neural network.``` should maybe be extended as adaptive learning and deep learning approaches seem to also be performant these days so we should not restrict ourselves to one approach at this point. i would just remove this sentence or else state that NN will be ```one``` of the approaches we will use.